aries-site/src/themes/aries.js

```diff
@@ -13,6 +13,12 @@ export const aries = deepMerge(hpe, {
       },
     },
   },
+  tabs: {
```

Comment:
Do we have an issue or PR for this work on grommet-theme-hpe?

Reply:
Yes, https://github.com/grommet/grommet-theme-hpe/pull/255

Reply:
nice can we take this out and run `yarn` to get latest stable theme to verify